dashboard/src/components/AppList/AppListItem.scss

```diff
@@ -0,0 +1,11 @@
+.DELETED {
```

Comment:
Minor comment and probably not worth changing.

Why the usage of uppercase?

Is it because it matches directly the status? Maybe we can transform it and still be consistent with other css class names?